CMake/ScmRevGen.cmake

```diff
@@ -50,3 +50,23 @@ configure_file(
 execute_process(COMMAND ${CMAKE_COMMAND} -E copy_if_different "${PROJECT_BINARY_DIR}/Source/Core/Common/scmrev.h.tmp" "${PROJECT_BINARY_DIR}/Source/Core/Common/scmrev.h")
 
 file(REMOVE "${PROJECT_BINARY_DIR}/Source/Core/Common/scmrev.h.tmp")
+
+if(APPLE)
+  configure_file(
+    "${PROJECT_SOURCE_DIR}/Source/Core/DolphinQt/Info.plist.in"
+    "${PROJECT_BINARY_DIR}/Source/Core/DolphinQt/Info.plist.tmp"
+  )
+
+  execute_process(COMMAND ${CMAKE_COMMAND} -E copy_if_different "${PROJECT_BINARY_DIR}/Source/Core/DolphinQt/Info.plist.tmp" "${PROJECT_BINARY_DIR}/Source/Core/DolphinQt/Info.plist")
+
+  file(REMOVE "${PROJECT_BINARY_DIR}/Source/Core/DolphinQt/Info.plist.tmp")
```

Comment:
This repeated 3-piece kinda irks me, does CMake have something like functions/methods that you can just call with parameters? Feels really error prone, even though you managed to not make any copy&paste mistakes (at least as far as I can tell)

Reply:
OK, I've refactored this part to use a function.